### ISSUE TITLE
Reminder every x amount of time

### ISSUE THREAD

**Issue by Fatih20** [ellie-commons/reminduck#55]
I wanted to make a reminder every two days. But currently it's not possible. The solution to this could be generalized to weeks and months too.

**Comment by matfantinel** [comment]
Agree, the option for "every x minutes" could be changed to allow for days, weeks and months.

In the meantime, "every x minutes" can work for you if you put 2880. 1440 minutes per day :smile: not pretty, but should work for now.

**Comment by Fatih20** [comment]
If you wanted to implement it, I'd like to suggest a UI change to make it better.

**Comment by matfantinel** [comment]
Sure, suggestion are welcome

**Comment by Fatih20** [comment]
![Reminduck](https://user-images.githubusercontent.com/51940383/88395642-9809a900-cdeb-11ea-97f4-2dbb87adfc23.png)
